MdePkg/Test/Mock/Include/GoogleTest/Protocol/MockSmmAccess2.h

```diff
@@ -0,0 +1,53 @@
+/** @file MockSmmAccess2.h
+  Declare mock SMM Access2 Protocol.
+

```

Comment:
Please add the copyright from the file being mocked. In this case the Intel Copyright: https://github.com/tianocore/edk2/blob/master/MdePkg/Include/Protocol/SmmAccess2.h#L14

Reply:
Please check the update.